Sources/Transport/Streams/Socks+Stream.swift

```diff
@@ -119,11 +119,13 @@ public final class TLSSocket<Socket: SocksCore.TCPInternetSocket> {
     public convenience init(
         mode: TLS.Mode,
         socket: Socket,
+        config: TLSConfig,
         certificates: TLS.Certificates = .none
```

Comment:
I think we should move Certificates into Config (mentioned on the vapor/tls thread)